docker-compose.yml

```diff
@@ -27,5 +27,17 @@ services:
       args:
         - NODE_PORT=$NODE_PORT
         - REACT_PORT=$REACT_PORT
+        - SERVER_NAME=$SERVER_NAME
     ports:
       - 80:80
+      - 443:443
+    volumes:
+      - ./certbot/www:/var/www/certbot/:ro
+      - ./certbot/conf/:/etc/nginx/ssl/:ro
```

Comment:
so here, `nginx` can "read-only" and `certbot` can "read/write"?

Reply:
Exact! `certbot` will write the certificates in the volume and `nginx` read them

Reply:
got it :+1: